### PR TITLE
Link to documentation in empty Raw Data view

### PR DESCRIPTION
**Pull request by wow-such-code**
Handles #737

### CODE REVIEW

**Comment by wow-such-code** [comment]
image

File: user-interface/src/main/java/life/qbic/datamanager/views/projects/project/rawdata/RawDataMain.java [hunk 1]
```diff
@@ -242,14 +245,22 @@ private Disclaimer createNoMeasurementsRegisteredDisclaimer() {
 
   private Disclaimer createNoRawDataRegisteredDisclaimer() {
     Disclaimer noRawDataRegistered = Disclaimer.createWithTitle(
-        "No Raw Data available",
-        "There is currently no raw data registered for the measurements within this experiment",
-        "Register Measurements");
+        "Register your Raw data first",
```
Comment (sven1103) [inline user-interface/src/main/java/life/qbic/datamanager/views/projects/project/rawdata/RawDataMain.java:248]: raw data (small letter)

File: user-interface/src/main/java/life/qbic/datamanager/views/projects/project/rawdata/RawDataMain.java [hunk 2]
```diff
@@ -242,14 +245,22 @@ private Disclaimer createNoMeasurementsRegisteredDisclaimer() {
...
+        "Raw data should be registered before you can view and download Raw data files.\n"
+            + "You can refer to our documentation to register Raw data for your measurements.",
```
Comment (sven1103) [inline user-interface/src/main/java/life/qbic/datamanager/views/projects/project/rawdata/RawDataMain.java:250]: raw data

**Review by sven1103 — CHANGES_REQUESTED** [review]
Only one typo, then I will approve

**Review by sven1103 — APPROVED** [review]
Nice 1!

**Pull request merged**